Mathlib/LinearAlgebra/LinearIndependent.lean

```diff
@@ -518,8 +518,7 @@ theorem linearIndependent_iUnion_of_directed {η : Type*} {s : η → Set M} (hs
     (h : ∀ i, LinearIndependent R (fun x => x : s i → M)) :
     LinearIndependent R (fun x => x : (⋃ i, s i) → M) := by
   by_cases hη : Nonempty η
```

Comment:
You will be able to golf further by doing `cases isEmpty_or_nonempty η` instead of that `by_cases

Reply:
Ok, but I would definitely say that this is out of scope for this PR!